make.ps1

```diff
@@ -4,7 +4,7 @@ Param(
     [Parameter(Position=1)]
     [String] $target = "release",
     [String] $configuration = "Release",
-    [String[]] $frameworks=@('net45','netcoreapp2.0','netcoreapp2.1'),
+    [String[]] $frameworks=@('net45','netcoreapp2.0','netcoreapp2.1', 'netstandard2.0'),
```

Comment:
This list is only used for testing and they're not being build against `netstandard2.0` in this PR so we shouldn't include this.

Reply:
Done